aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -263,12 +294,86 @@ class QConvPackWeightInt8 final : public c10::OperatorKernel {
         "Didn't find engine for operation quantized::conv_prepack ",
         toString(ctx.qEngine()));
   }
+
+ private:
+#ifdef USE_FBGEMM
+  static fbgemm::conv_param_t<kSpatialDim> MakeFbgemmConvParam(
+      int C,
+      int M,
+      int group,
+      const std::vector<int>& kernel,
+      const torch::List<int64_t>& stride,
+      const torch::List<int64_t>& padding,
+      const torch::List<int64_t>& dilation);
+#endif // USE_FBGEMM
 };
 
-static auto registry = c10::RegisterOperators().op(
-    "quantized::conv_prepack",
-    c10::RegisterOperators::options().kernel<QConvPackWeightInt8>(
-        TensorTypeId::QuantizedCPUTensorId));
+#ifdef USE_FBGEMM
+
+template <>
+fbgemm::conv_param_t<2> QConvPackWeightInt8<2>::MakeFbgemmConvParam(
+    int C,
+    int M,
+    int groups,
+    const std::vector<int>& kernel,
+    const torch::List<int64_t>& stride,
+    const torch::List<int64_t>& padding,
+    const torch::List<int64_t>& dilation) {
+  return fbgemm::conv_param_t<2>(
```

Comment:
for this `conv_param_t` invocation and the one below, can we getp aram name comments (e.g. like https://github.com/pytorch/pytorch/blob/d83389d327157b04ea74278c1026e6e370ba4001/aten/src/ATen/native/quantized/cpu/qlinear.cpp#L170)

Reply:
Done.